src/Tiqr/TiqrServiceInterface.php

```diff
@@ -17,23 +18,27 @@
 
 namespace App\Tiqr;
 
+use App\Exception\TiqrServerRuntimeException;
+use App\Tiqr\Legacy\TiqrService;
 use App\Tiqr\Response\AuthenticationResponse;
+use Exception;
+use phpDocumentor\Reflection\Types\Boolean;
```

Comment:
This `Boolean` import is not used, neither are the `Exception` and `App\Tiqr\Legacy\TiqrService`.

IDE tip: PHPStorm makes quick work of this by using the Optimize imports feature. You can find it in the main menu: `Code` => `Optimize imports`. You can run it on the `scr` folder to fix this for all files in one fell swoop

Reply:
Done. Good tip!